index.bs

```diff
@@ -900,77 +900,102 @@ See [[#naming-is-hard]] for general (cross-language) advice on naming.
 
 <h3 id="js-only">Web APIs are for JavaScript</h3>
 
-The language that web APIs are meant to be used in, and specified for, is JavaScript (also known as
-[[!ECMASCRIPT]]). They are not language-agnostic, and are not meant to be.
+When designing imperative APIs for the Web,
+use JavaScript.
+In particular,
+you can freely rely upon language-specific semantics and conventions,
+with no need to keep things generalized.
+
+<div class="example">
+For example,
+the {{CustomElementRegistry.define()}} method takes a reference to
+a <a href="https://tc39.es/ecma262/#sec-static-semantics-constructormethod">Constructor Method</a>.
```

Comment:
Is https://tc39.es/ecma262/ the proper link to refer to?

Reply:
I mean like the official one as it refers to a Spanish site (.es)

Reply:
Apparently this is the official one.